lib/services/debug-service.ts

```diff
@@ -92,6 +93,16 @@ export class DebugService extends EventEmitter implements IDebugService {
 		connectionErrorHandler = connectionErrorHandler.bind(this);
 		platformDebugService.on(CONNECTION_ERROR_EVENT_NAME, connectionErrorHandler);
 	}
+
+	private getDebugInformation(fullUrl: string): IDebugInformation {
+		const parseQueryString = true;
+		const wsQueryParam = parse(fullUrl, parseQueryString).query.ws;
+		const hostPortSplit = wsQueryParam && wsQueryParam.split(":");
+		return {
+			url: fullUrl,
+			port: hostPortSplit && +hostPortSplit[1]
```

Comment:
in case the wsQueryParam does not contain `:`, you'll end up with port = NaN, is this okay?

Reply:
I think so - in case the `ws` query param isn't in the correct format I can't really tell the port so a `null`-like value seems okay